packages/kotti-ui/source/props.ts

```diff
@@ -2,11 +2,20 @@ import { PropOptions, PropType } from '@vue/composition-api'
 import { cloneDeep } from 'lodash'
 import { z } from 'zod'
 
-import { MapValueType } from './types/utilities'
-
 const DEBUG_MAKE_PROPS = false as const // enable to print debug log
 const DEBUG_WALK_SHAPE = false as const // enable to print debug log
 
+/**
+ * This type is not exported directly by @vue/composition-api
+ */
+type VuePropConstructor = Exclude<PropType<unknown>, Array<unknown>>
```

Comment:
wouldn't it be `Array<PropType<unknown>>`?

Reply:
It’s possible to do that, but that wouldn’t make this type any better and it’s also weird to exclude `Array<Array<PropConstructor>>` as well. I’m just excluding anything array-like from this type which I think is pretty clear.